localstack/services/lambda_/packages.py

```diff
@@ -76,51 +67,6 @@ def _install(self, target: InstallTarget) -> None:
         os.chmod(install_location, mode=st.st_mode | stat.S_IXUSR | stat.S_IXGRP | stat.S_IXOTH)
 
 
-# TODO[LambdaV1]: Remove
-class LambdaGoRuntimePackage(Package):
-    def __init__(self, default_version: str = GO_RUNTIME_VERSION):
-        super().__init__(name="LambdaGo", default_version=default_version)
-
-    def get_versions(self) -> List[str]:
-        return [GO_RUNTIME_VERSION]
-
-    def _get_installer(self, version: str) -> PackageInstaller:
-        return LambdaGoRuntimePackageInstaller(name="lambda-go-runtime", version=version)
-
-
-# TODO[LambdaV1]: Remove
-class LambdaGoRuntimePackageInstaller(ArchiveDownloadAndExtractInstaller):
-    def _get_download_url(self) -> str:
-        system = platform.system().lower()
-        arch = get_arch()
-
-        if system not in ["linux"]:
-            raise SystemNotSupportedException(f"Unsupported os {system} for lambda-go-runtime")
-        if arch not in ["amd64", "arm64"]:
-            raise SystemNotSupportedException(f"Unsupported arch {arch} for lambda-go-runtime")
-
-        return GO_RUNTIME_DOWNLOAD_URL_TEMPLATE.format(
-            version=GO_RUNTIME_VERSION,
-            os=system,
-            arch=arch,
-        )
-
-    def _get_install_marker_path(self, install_dir: str) -> str:
-        return os.path.join(install_dir, "aws-lambda-mock")
-
-    def _install(self, target: InstallTarget) -> None:
-        super()._install(target)
-
-        install_dir = self._get_install_dir(target)
-        install_location = self._get_install_marker_path(install_dir)
-        st = os.stat(install_location)
-        os.chmod(install_location, st.st_mode | stat.S_IXUSR | stat.S_IXGRP | stat.S_IXOTH)
-
-        go_lambda_mockserver = os.path.join(install_dir, "mockserver")
-        st = os.stat(go_lambda_mockserver)
-        os.chmod(go_lambda_mockserver, st.st_mode | stat.S_IXUSR | stat.S_IXGRP | stat.S_IXOTH)
-
-
 # TODO: replace usage in LocalStack tests with locally built Java jar and remove this unmaintained dependency.
 class LambdaJavaPackage(Package):
```

Comment:
This can be removed post v3 but requires updating the Java build infrastructure for some Java Lambda tests